cmd/tigerblood/main.go

```diff
@@ -5,59 +5,45 @@ import (
 	"fmt"
 	"github.com/DataDog/datadog-go/statsd"
 	"go.mozilla.org/tigerblood"
-	"gopkg.in/yaml.v2"
-	"io/ioutil"
 	"log"
 	"net/http"
-	"os"
 )
 
-func loadConfig(path string) (tigerblood.Config, error) {
-	var config tigerblood.Config
-	bytes, err := ioutil.ReadFile(path)
-	if err != nil {
-		return config, err
-	}
-	err = yaml.Unmarshal(bytes, &config)
-	return config, err
-}
-
 func main() {
 	configFile := flag.String("config-file", "config.yml", "Path to the YAML config file")
 	flag.Parse()
-	config, err := loadConfig(*configFile)
+	config, err := tigerblood.LoadConfigFromPath(*configFile)
 	if err != nil {
 		log.Println("Error loading config file:", err)
 	}
-	dsn, found := os.LookupEnv("TIGERBLOOD_DSN")
-	if !found {
+	if config.DatabaseDsn == "" {
 		log.Println("No database DSN found")
 	}
-	db, err := tigerblood.NewDB(dsn)
+	db, err := tigerblood.NewDB(config.DatabaseDsn)
 	if err != nil {
 		log.Fatal(fmt.Errorf("Could not connect to the database: %s", err))
 	}
 	db.SetMaxOpenConns(80)
```

Comment:
80 can be the default, but this should be configurable. Can be in this PR or another.